src/app/forms/custom-sections/required-standard/required-standard.component.html

```diff
@@ -1 +1,43 @@
-<h1>I am a required standard</h1>
+<ng-template [ngTemplateOutlet]="isEditing ? editRs : viewRs" [ngTemplateOutletContext]="{ data: form }"></ng-template>
+
+<ng-template #viewRs let-data="data">
+  <h1>View me</h1>
+</ng-template>
+
+<ng-template #editRs let-data="data">
+  <ng-container *ngIf="requiredStandard">
+    <div class="govuk-grid-row govuk-!-margin-bottom-5">
+      <div class="govuk-grid-column-four-fifths">
+        <h1 class="govuk-heading-l">Required Standard Details</h1>
+        <h2 class="govuk-heading-s">{{ requiredStandard.refCalculation }} {{ requiredStandard.sectionDescription }}</h2>
+        <h3 class="govuk-heading-s">{{ requiredStandard.requiredStandard }}</h3>
```

Comment:
Just to bring the designs more inline with defects
```suggestion
        <span class="govuk-caption-m">{{ requiredStandard.sectionDescription }}</span>
        <h2 class="govuk-heading-s">{{ requiredStandard.refCalculation }} - {{ requiredStandard.sectionDescription }}: {{ requiredStandard.requiredStandard }}
        </h2>
```